frontend/app/src/components/price-manager/latest/LatestPriceTable.vue

```diff
@@ -66,6 +66,7 @@
 <script setup lang="ts">
 import { BigNumber } from '@rotki/common';
 import { NotificationPayload, Severity } from '@rotki/common/lib/messages';
+import { flatten } from 'lodash';
```

Comment:
better import flatten from lodash/flatten instead.

At least from what I remember from the documentation directly importing from lodash doesn't play well with tree-shaking